src/wristpy/processing/analytics.py

```diff
@@ -308,41 +308,80 @@ def _compute_abs_diff_mean_anglez(
             measurements=absolute_diff, time=anglez_epoch1.time[1:]
         )
 
-    def _find_periods(
-        self, window_measurement: models.Measurement
-    ) -> List[Tuple[datetime, datetime]]:
-        """Find periods where window_measurement is equal to 1.
 
-        This is a helper function for the _find_onset_wakeup_times to
-        find periods where either the spt_window or sib_periods are equal to 1.
+def _find_periods(
+    window_measurement: models.Measurement,
+) -> List[Tuple[datetime, datetime]]:
```

Comment:
datetime is imported from where? :)

Reply:
I switched it everywhere so now at the import level it is just
`import datetime`

and for the class definition it is `datetime.datetime`. Is this correct? the datetime.datetime confuses me aha